packages/nx/src/utils/git-utils.ts

```diff
@@ -0,0 +1,69 @@
+import { execSync } from 'child_process';
+
+export function getGithubSlugOrNull(): string | null {
+  const gitRemote = execSync('git remote -v').toString();
```

Comment:
Could we use `git ls-remote --get-url` instead? We wouldn't have to parse anything then.

Reply:
I want to make sure we're getting the right remote, this is why I'm listing all.